.docker.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+set -x
+
+env
+storage-ng-travis-cpp && bash <(curl -s https://codecov.io/bash)
```

Comment:
To me it looks very dangerous to automatically execute code downloaded from an untrusted site.